[Repository] Documentation updated (READMES and folders). Closes #2 (#3)
Hernancano98/Project-Book: README.md

# **Project-Book**

> This is a collaborative repository about a **collection of projects** that use **different technologies**. Each collaborator uploads his own code in the corresponding folders with a complete description of his work.

Reviewer: ```suggestion
This is a collaborative repository about a **collection of projects** that use **different technologies**. Each collaborator uploads his own code to the corresponding directory with a complete description of his work.
```

+ Languages:
  - [Java](./Java/README.md)
  - [JavaScript](./JavaScript/README.md)

>   - [C#](./Csharp/README.md)

Reviewer 2: Please, change the name of the directory to C#

  - [Python](./Python/README.md)
+ Projects:

>   - [REST API](./Java/HernanCano/API_REST/README.md) - Hernán Cano

Reviewer: Decide if you are going to call it REST API or API REST, you shouldn't call the link REST API and the directory API_REST

```suggestion
  - [REST API](./Java/HernanCano/API_REST/README.md)
```
